rholang/src/main/bnfc/rholang_mercury.cf

```diff
@@ -51,13 +51,13 @@ PIfElse.      Proc1 ::= "if" "(" Proc ")" Proc2 "else" Proc1 ;
 PNew.         Proc1 ::= "new" [NameDecl] "in" Proc1 ;
 PPar.         Proc  ::= Proc "|" Proc1 ;
 
-separator nonempty Proc "," ;
+separator Proc "," ;
```

Comment:
I don't see any usage below of having an empty terminator for a Proc. Are we intentionally adding this as a feature of the language?

Reply:
It's to make the empty list `[]` compile. Removing nonempty from proc and name separators was @guardbotmk3 's suggestion